components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1745,6 +1767,14 @@ interface ZCashWalletService {
       (ZCashAddressValidationResult result);
   ShieldFunds(string network_id, AccountId account_id) =>
       (string? tx_id, string? error_message);
+
+  IsAccountShielded(AccountId account_id) => (bool result);
+  MakeAccountShielded(AccountId account_id) => (string? error_message);
+  GetAccountShieldBirthday(AccountId account_id) => (ZCashAccountShieldBirthday? birthday, string? error_message);
+
+  GetShieldSyncStatus(AccountId account_id) => (ZCashShieldSyncStatus? status, string? error);
+  StartShieldSync(AccountId account_id, pending_remote<ZCashSyncObserver> observer) => (string? error_message);
+  StopShieldSync(AccountId account_id) => (string? error_message);
```

Comment:
It seems it makes sense to return these
```
IsAccountShielded
GetAccountShieldBirthday
GetShieldSyncStatus
```
as fields GetZCashAccountInfo() -> ZCashAccountInfo

Reply:
Done